cmd/format-erasure.go

```diff
@@ -387,7 +374,9 @@ func saveFormatErasure(disk StorageAPI, format *formatErasureV3, heal bool) erro
 
 	disk.SetDiskID(diskID)
 	if heal {
-		return saveHealingTracker(disk, diskID)
+		ctx := context.Background()
+		ht := newHealingTracker(disk)
+		return ht.save(ctx)
```

Comment:
But let's just ignore this error and only log it ? we are formatting here.. I hope the upper layer will just retry this indefinitely :)

Reply:
It will. That is the design.